staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -40,6 +40,16 @@ var (
 		[]string{"verb", "host"},
 	)
 
+	requestSize = k8smetrics.NewHistogramVec(
+		&k8smetrics.HistogramOpts{
+			Name: "rest_client_request_size_bytes",
```

Comment:
hmm, I think response is a better name

Reply:
yeah, we should measure both , there are cases that the problems can be related to POST or GET with huge payloads,